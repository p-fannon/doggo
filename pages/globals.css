@tailwind base;
@tailwind components;
@tailwind utilities;

:root {
  --foreground-rgb: 0, 0, 0;
  --background-start-rgb: 255, 204, 220;
  --background-end-rgb: 255, 255, 255;
}

@media (prefers-color-scheme: dark) {
  :root {
    --foreground-rgb: 255, 255, 255;
    --background-start-rgb: 102, 51, 102;
      --background-end-rgb: 102, 51, 102;
  }
}

body {
  color: rgb(var(--foreground-rgb));
  background: linear-gradient(
      to bottom,
      transparent,
      rgb(var(--background-end-rgb))
    )
    rgb(var(--background-start-rgb));
}

@layer utilities {
  .text-balance {
    text-wrap: balance;
  }
}

header {
  background-color: rgba(255, 204, 220, 1);
}

footer {
  background-color: rgba(255, 255, 255, 1);
}

@media (prefers-color-scheme: dark) {
  header {
      background-color: rgba(102, 51, 102, 1);
    }
  
    footer {
      background-color: rgba(102, 51, 102, 1);
  }
}