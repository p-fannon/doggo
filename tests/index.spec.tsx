import HomePage from "../pages/index";
import { describe, it } from '@jest/globals'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'

describe('HomePage', () => {
    it('renders successfully', () => {
        const { container } = render(<HomePage />)
        expect(container).toMatchSnapshot()
        expect(screen.getByRole('button')).toHaveTextContent('Fetch')
    })
})
