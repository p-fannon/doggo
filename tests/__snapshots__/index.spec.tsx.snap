// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders successfully 1`] = `
<div>
  <div
    class="flex flex-col mt-6 justify-center items-center"
  >
    <button
      class="chakra-button css-4xx2wk"
      type="button"
    >
      Fetch
    </button>
  </div>
</div>
`;
